src/titlepane/example/index.ts

```diff
@@ -0,0 +1,123 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { StatefulMixin } from '@dojo/widget-core/mixins/Stateful';
+import { v, w } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+
+import TitlePane from '../TitlePane';
+import dojoTheme from '../../themes/dojo/theme';
+
+export class App extends StatefulMixin(WidgetBase)<WidgetProperties> {
+	private _theme: {};
+
+	themeChange(event: Event) {
+		const checked = (<HTMLInputElement> event.target).checked;
```

Comment:
can type as `event: TypedTargetEvent<HTMLInputElement>`, and then you don't need to cast `event.target`